Documentation/operations/upgrade.rst

```diff
@@ -435,6 +450,9 @@ Helm Options
   domain socket will be removed and the Kubernetes peer-svc mandatory when
   Hubble Relay is enabled. Users overriding ``hubble.peerService.enabled`` to
   ``false`` are encouraged to migrate to ``true`` with the Cilium v1.13 release.
+* ``bpfClockProbe`` has been set to ``false`` everywhere to avoid interrupting
+  existing connections. It can still be manually set to ``true`` if the
+  ``bpf-clock-probe`` behavior is desired.
```

Comment:
I'm confused about this. `bpfClockProbe` is the helm option, shouldn't it be "`bpfClockProbe` behavior is desired."?

Reply:
Reworded as mentioned. PTAL